app/views/devise/shared/_oauthlogin.html.slim

```diff
@@ -0,0 +1,18 @@
+- if devise_mapping.omniauthable?
```

Comment:
@manfromjupyter Title and screen reader only text is probably overkill here, but I put in both.  I want something for the sighted user in a hover-text, so what do you think is best here.

Reply:
Title doesn't help most if not any screenreaders in my experience. Some are smarter than others, but for JAWS, the most popular one of them all, the way you have it is perfect.